build.sh

```diff
@@ -29,4 +29,4 @@ go build -a -ldflags="-s -w" -installsuffix cgo -o dist/mouthful main.go
 chmod +x dist/mouthful
 
 # copy over config
-cp ./config.json dist/data/config.json
```

Comment:
Could we default to dist/data/config.json? We want the sub folder so that we can mount a volume for docker.